iothub/device/src/ModuleClient.cs

```diff
@@ -845,30 +860,20 @@ private async Task<MethodResponse> InvokeMethodAsync(Uri uri, MethodRequest meth
                         SslProtocols = TlsVersions.Instance.Preferred,
                     };
 #else
-                    httpClientHandler = new WebRequestHandler();
+#pragma warning disable CA2000 // Dispose objects before losing scope
+                    // This object is given to an HTTP client to use, so it can't be disposed yet
+                    var httpMessageHandler = new WebRequestHandler();
```

Comment:
Is this being disposed when the module client is disposed?

Reply:
It is now. Good catch